slack_notifications_lambda.tf

```diff
@@ -0,0 +1,29 @@
+locals {
+  notifications_lambda_name                   = "${local.environment}-slack-notifications"
+}
+
+module "slack_notifications_lambda" {
+  source        = "git::https://github.com/nationalarchives/da-terraform-modules//lambda"
+  function_name = local.notifications_lambda_name
+  handler       = "lambda_function.lambda_handler"
+
+  policies = {
+    "${local.notifications_lambda_name}-policy" = templatefile("./templates/iam_policy/slack_notifications_policy.json.tpl", {
+      secrets_manager_secret_arn   = "arn:aws:secretsmanager:eu-west-2:${var.dp_account_number}:secret:slack_webhook_url-xrHr13",
```

Comment:
This will need to change to use the ssm parameter arn. You shouldn't need to construct the arn, you should be able to do `aws_ssm_parameter.slack_webhook.arn`